### ISSUE TITLE
Extra "<p>" is generated for an empty line outside of comments

### ISSUE THREAD

**Issue by jpsacha** [bytedeco/javacpp#475]
When parsing C++ source extra `<p>` is generated in Java code outside of the comments. A fragment of the source C++ code (including original indentation):
```cpp
        /*@}*/
        /*@}*/

        /**
         *  @addtogroup Container_h Container Class
         */
        /*@{*/

        //*************************************************************
        // V A L U E _ V E C T O R
        //*************************************************************

        /**
         * @brief A vector of IValue pointers
         */
        class SPINNAKER_API value_vector
```

The Java code generated by JavaCPP (1.5.6-SNAPSHOT) resulted in:

```java
        /*@}*/
        /*@}*/

        /**
         *  \addtogroup Container_h Container Class
         */
        /*@{*/

        //*************************************************************
        // V A L U E _ V E C T O R
        //*************************************************************
        <p>
        /**
         * \brief A vector of IValue pointers
         */
        @Namespace("Spinnaker::GenApi") @NoOffset @Properties(inherit = org.bytedeco.spinnaker.presets.Spinnaker.class)
public class value_vector extends Pointer {
```

Notice that there is extra `<p>` outside of the comments.

**Comment by saudet** [comment]
Fixed in commit https://github.com/bytedeco/javacpp/commit/343045d9c3f01856691c9b8576cbc3e98ee6e369. Thanks for reporting!

**Comment by jpsacha** [comment]
The fix works for me

**Comment by saudet** [comment]
Fix released in JavaCPP 1.5.6. Enjoy!
